tests/baselines/reference/ambientShorthand_reExport.js

```diff
@@ -20,11 +20,19 @@ x($);
 "use strict";
 exports.__esModule = true;
 var jquery_1 = require("jquery");
-exports.x = jquery_1.x;
+Object.defineProperty(exports, "x", { enumerable: true, get: () => jquery_1.x });
 //// [reExportAll.js]
 "use strict";
 function __export(m) {

```

Comment:
`tslib` will also need to be updated

Reply:
Okay, I made the change there as well: https://github.com/microsoft/tslib/pull/78